query/search/search.go

```diff
@@ -119,10 +120,29 @@ func (q *Query) ToSearchGroupBy() string {
 	return groupBy
 }
 
+func (q *Query) ToSearchVector() string {
+	if len(q.VectorS.VectorF) == 0 {
+		return ""
```

Comment:
Will returning an empty string have any effect on the search query?

Reply:
The caller is checking it and not filling in the params if it is an empty string.